src/browser/web-view.js

```diff
@@ -327,7 +329,7 @@ define((require, exports, module) => {
       display: 'none'
     },
     passive: {
-      visibility: 'collapsed'
+      visibility: 'collapse'
```

Comment:
Nit: `collapse` is an obscure property value meant for tables. We're taking advantage of the fact that it falls back to `hidden` on non-table elements. Better to use `hidden`. https://developer.mozilla.org/en-US/docs/Web/CSS/visibility